routers/repo/release.go

```diff
@@ -260,6 +270,14 @@ func EditReleasePost(ctx *context.Context, form auth.EditReleaseForm) {
 		return
 	}
 
+	tag, err := ctx.Repo.GitRepo.GetTag(rel.TagName)
```

Comment:
After doing more tests, you are right, this block is not necessary on `EditReleasePost` function. I will remove this today.